spec/linter-eslint-spec.js

```diff
@@ -128,21 +140,70 @@ describe('The eslint provider for Linter', () => {
     })
   })
 
-  describe('Fix errors when saved', () => {
+  describe('Fix errors', () => {
+    let editor
+    let doneCheckingFixes
+    let tempFixtureDir
+    let tempFixturePath
+    let tempConfigPath
+
     beforeEach(() => {
-      atom.config.set('linter-eslint.fixOnSave', true)
+      waitsForPromise(() => {
+        tempFixtureDir = fs.mkdtempSync(tmpdir() + path.sep)
+        tempFixturePath = `${tempFixtureDir}${path.sep}fixed.js`
```

Comment:
Any particular reason not to use `path.join`? This works just fine as is, just the rest of the file uses `path.join`.